src/main/java/com/google/devtools/build/lib/starlarkbuildapi/repository/StarlarkOverrideApi.java

```diff
@@ -0,0 +1,13 @@
+package com.google.devtools.build.lib.starlarkbuildapi.repository;
+
+import com.google.devtools.build.docgen.annot.DocCategory;
+import net.starlark.java.annot.StarlarkBuiltin;
+import net.starlark.java.eval.StarlarkValue;
+
+@StarlarkBuiltin(
+    name = "override",
+    category = DocCategory.BUILTIN,
+    doc = "Overrides some information about a Bazel module dependency.")
+public interface StarlarkOverrideApi extends StarlarkValue {
```

Comment:
We don't need the `*Api` interface anymore, thanks to Ivo's work. Let's not introduce more of them?